mkdocs.yml

```diff
@@ -7,7 +7,6 @@ nav:
     - Database schema: schema.md
     - Schema loader: schema-loader.md
 - Design: design.md
-- Jepsen: jepsen.md
```

Comment:
Let's create a new jepsen.md to point to the new repo.

Reply:
Thank you. Added.